manifests/crd/acceleratorprofiles.opendatahub.io.crd.yaml

```diff
@@ -28,27 +28,36 @@ spec:
               properties:
                 displayName:
                   type: string
+                  description: "The display name of the accelerator profile."
                 enabled:
                   type: boolean
+                  description: "Indicates whether the accelerator profile is available for new resources."
                 identifier:
                   type: string
+                  description: "The resource identifier of the accelerator device. Example: nvidia.com/gpu"
                 description:
                   type: string
+                  description: "A short description of the accelerator profile."
                 tolerations:
                   type: array
+                  description: "A Kubernetes toleration value that is added to resources using this accelerator profile."
                   items:
                     type: object
                     required:
                       - key
                     properties:
                       key:
                         type: string
+                        description: "The key of the toleration."
                       operator:
                         type: string
+                        description: "The operator of the toleration."
                       value:
                         type: string
+                        description: "The value of the toleration."
                       effect:
                         type: string
+                        description: "The effect of the toleration."
                       tolerationSeconds:
                         type: integer
-   
\ No newline at end of file
+                        description: "The duration in seconds of the toleration."
```

Comment:
Hmmm... so this is the [k8s docs on Tolerations](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#toleration-v1-core). I wonder if this is a bit too simple "[property name] of the toleration" doesn't bring a lot of value.

I don't really want to copy & paste k8s documentation, if a property pivots to support additional values, we'd need to update our docs too. I wonder if we can take a synopsis of each definition and give a very basic representation here 🤔 

"Effect indicates the taint effect to match. Empty means match all taint effects." is simple enough for `effect`, for instance.

Reply:
i updated the PR